python/cac_tripplanner/templates/partials/destination-card.html

```diff
@@ -0,0 +1,9 @@
+<a href="{% url 'place-detail' pk=destination.pk %}" class="preview-card-link">
+    <div class="preview-card">
+        <img class="preview-photo" src="{{ destination.wide_image.url }}" width="308" height="204" alt="{{ destination.name }}" />
+        <div class="preview-details">
+            <h2>{{ destination.name }}</h2>
+            <p class="lede">{{ destination.description|striptags|truncatewords:20 }}</p>
```

Comment:
Yeah, I like this. If we need to do anything more complex in future I think it would make sense to move it out.